docs/platform/howto/btar/manage-backup-to-another-region.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Manage a cross-region backup
+sidebar_label: Manage cross-region backup
+pro: true
+---
+
+For a service that has the [backup to another region (BTAR) feature](/docs/platform/concepts/backup-to-another-region) enabled, you can check the service backup status, change the backup region, monitor the replication lag, fork and restore using [the cross-region backup](/docs/platform/concepts/backup-to-another-region), or migrate to another cloud or region.
+
+:::important
+Backup to another region (BTAR) is available on
+`[Pro Platform](/docs/platform/concepts/pro-platform)`.
+:::
+
+## Prerequisites
+
+You have at least one
+[Aiven service with BTAR enabled](/docs/platform/howto/btar/enable-backup-to-another-region).
+
+## Change a backup region
+
+1. Log in to the [Aiven Console](https://console.aiven.io/).
+1. From the **Services** view, select an Aiven service on which you'd like to enable BTAR.
+1. On your service's page, select **Backups** from the sidebar.
+1. On the **Backups** page, select the actions (**...**) menu > **Secondary backup
+   location**.
+1. In the **Edit secondary backup location** window, use the **Secondary location**
+   dropdown menu to select a region for your additional backup. Confirm your choice by
+   selecting **Save**.
+
+:::important
+Think carefully before you decide to change a backup region. Since each change of a backup
+region triggers a new base backup, you can change a backup region only infrequently.
+:::
+
+## Monitor a service with BTAR
+
+There are a few things you may want to check for your Aiven service in the context for
+BTAR:
+
+- What is the status of a secondary backup?
+
+  - Does your service has a backup in another region?
+  - What is the target region of the secondary backup?
+
+- What is the replication lag between data availability in the primary region and the
+  secondary region?
+
+### Check BTAR status in the console
+
+To check out the availability, the status, and the target region of a secondary (BTAR)
+backup in the [Aiven Console](https://console.aiven.io/), navigate to your
+**service's page** > **Backups** view > **Secondary backup location** column.
+
+### Check replication lag with API
+
+To check out the target region and the replication lag for a secondary (BTAR) backup of
```

Comment:
Nit: It's `check` in the title, `check out` in the text (which is rather informal). What about `determine` to avoid phrasal verbs and ensure uniformity?